$NetBSD: patch-aa,v 1.6 2023/10/17 19:24:04 triaxx Exp $

Remove environment variables which define non-portable programs, macros or options.

--- Makefile.orig	2023-02-23 15:58:45.000000000 +0000
+++ Makefile
@@ -3,10 +3,6 @@
 # (C) 1997-2007 Nelson Murilo, Pangeia Informatica, AMS Foundation and others.
 #
 
-CC       ?= cc
-CFLAGS	 += -DHAVE_LASTLOG_H
-STATIC   = -static
-
 ###
 ### Solaris 2.x
 ###
